src/schema/v2/partner/partner.ts

```diff
@@ -342,7 +342,12 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
                 ...connectionFromArraySlice(body, args, {
                   arrayLength: body.length,
                   sliceStart: 0,
-                  resolveNode: (node) => node.artist,
+                  resolveNode: (node) => {
+                    return {
+                      ...node.artist,
+                      hide_in_presentation_mode: node.hide_in_presentation_mode,
```

Comment:
Why put this onto the node (artist) when its a partner-artist concern and can live on the edge like the other fields?

Reply:
Ah totally, that would work 👍 I haven't placed much stuff on edges before, but that's where it should go. This felt a little wonky. (will fix)

Reply:
wow this is WAY better, thanks for the flag @mzikherman

Reply:
Yay! It makes a lot of sense - join models are almost always an implementation detail that shouldn't be first-class.

But like many schema design cases - it can be a bit of a judgement call, there's no mathematically right or wrong way.

h/t the Apollo blog post that initially mentioned this pattern which I really like (whole blog post is 👍 ) - https://www.apollographql.com/blog/explaining-graphql-connections#data-on-the-edges